src/pages/lists/status-list/index.tsx

```diff
@@ -1,78 +1,371 @@
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import Chevron from '@material-ui/icons/ChevronRight';
+import NotificationsActiveIcon from '@material-ui/icons/NotificationsActive';
 import NotificationIcon from '@material-ui/icons/Notifications';
-import WarningIcon from '@material-ui/icons/Warning';
+import HelpIcon from '@material-ui/icons/Help';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
 import { useDispatch } from 'react-redux';
 import { TOGGLE_DRAWER } from '../../../redux/actions';
-import { InfoListItem, ListItemTag, InfoListItemProps } from '@pxblue/react-components';
+import { InfoListItem, ListItemTag, InfoListItemProps, Spacer } from '@pxblue/react-components';
 import * as colors from '@pxblue/colors';
+import { Maintenance } from '@pxblue/icons-mui';
+import clsx from 'clsx';
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme: Theme) => ({
     appbarRoot: {
         padding: 0,
     },
     toolbarGutters: {
-        padding: '0 16px',
+        padding: `0 ${theme.spacing(2)}px`,
+        display: 'flex',
+        justifyContent: 'space-between',
+    },
+    listItemTextWithoutIcon: {
+        marginLeft: -theme.spacing(7),
+    },
+    listItemText: {
+        marginLeft: '0px',
+    },
+    accordionContainer: {
+        maxWidth: 768,
+        paddingTop: theme.spacing(3),
+        margin: '0 auto',
+
+        [theme.breakpoints.down('sm')]: {
+            margin: `0 0 ${theme.spacing(3)}px 0`,
+            paddingTop: 0,
+        },
+    },
+    accordionRoot: {
+        marginBottom: theme.spacing(2),
+        borderRadius: '4px',
+        '&:before': {
+            display: 'none',
+        },
+        '& .MuiAccordionSummary-root': {
+            height: '48px',
+            minHeight: '48px',
+            '&.Mui-expanded': {
+                borderBottom: `1px solid ${theme.palette.divider}`,
+            },
+        },
+        [theme.breakpoints.down('sm')]: {
+            maxWidth: '100%',
+            boxShadow: 'none',
+            borderRadius: 0,
+            '&:before': {
+                display: 'none',
+            },
+            '& .MuiAccordionSummary-root': {
+                height: '48px',
+                minHeight: '48px',
+                '&.Mui-expanded': {
+                    borderBottom: `1px solid ${theme.palette.divider}`,
+                    margin: 0,
+                },
+            },
+        },
+    },
+    accordionDetails: {
+        display: 'block',
+        padding: 0,
+    },
+    toolbarRightContent: {
+        display: 'flex',
+        flexDirection: 'row',
+    },
+    listItemTitle: {
+        display: 'flex',
+        alignItems: 'center',
+        color: colors.black[500],
+        fontWeight: 400,
+        marginLeft: theme.spacing(4),
+    },
+    listItemTitleWithoutTimeStamp: {
+        marginLeft: 'auto',
+    },
+    station: {
+        fontSize: 14,
+        fontWeight: 400,
+        marginLeft: theme.spacing(4),
```

Comment:
see the previous comment about spacing for title, subtitle, and info.

Reply:
Done